### PR TITLE
New version: SnapmakerDevTeam.SnapmakerLuban version 4.1.1

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/37608)

### CODE REVIEW

**Review by ItzLevvie — DISMISSED** [review]

File: manifests/s/SnapmakerDevTeam/SnapmakerLuban/4.1.1/SnapmakerDevTeam.SnapmakerLuban.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: SnapmakerDevTeam.SnapmakerLuban
+PackageVersion: 4.1.1
+PackageLocale: en-US
+Publisher: Snapmaker Dev Team
+PublisherUrl: https://snapmaker.com/
+PublisherSupportUrl: https://snapmaker.com/support
```
Comment (ItzLevvie) [inline manifests/s/SnapmakerDevTeam/SnapmakerLuban/4.1.1/SnapmakerDevTeam.SnapmakerLuban.locale.en-US.yaml:9]:
```suggestion
#PublisherSupportUrl: https://snapmaker.com/support
```
ZenDesk URLs are blocked in Azure DevOps.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/s/SnapmakerDevTeam/SnapmakerLuban/4.1.1/SnapmakerDevTeam.SnapmakerLuban.locale.en-US.yaml (hunk 1 above)
Comment (Trenly) [reply manifests/s/SnapmakerDevTeam/SnapmakerLuban/4.1.1/SnapmakerDevTeam.SnapmakerLuban.locale.en-US.yaml]: Ugh. Thats really annoying

**Review by ItzLevvie — APPROVED** [review]

File: manifests/s/SnapmakerDevTeam/SnapmakerLuban/4.1.1/SnapmakerDevTeam.SnapmakerLuban.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
```
Comment (ItzLevvie) [inline manifests/s/SnapmakerDevTeam/SnapmakerLuban/4.1.1/SnapmakerDevTeam.SnapmakerLuban.installer.yaml:16]:
```suggestion
- Architecture: x86
```

File: manifests/s/SnapmakerDevTeam/SnapmakerLuban/4.1.1/SnapmakerDevTeam.SnapmakerLuban.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  InstallerUrl: https://github.com/Snapmaker/Luban/releases/download/v4.1.1/Snapmaker-Luban-4.1.1-win-ia32.exe
+  InstallerSha256: DDE1BB71C6E0F23B3350B8D1AB4A35A30DEAF1BD1C4119A24C365208BD2D73AF
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/s/SnapmakerDevTeam/SnapmakerLuban/4.1.1/SnapmakerDevTeam.SnapmakerLuban.installer.yaml:19]:
```suggestion
- Architecture: x64
```

File: manifests/s/SnapmakerDevTeam/SnapmakerLuban/4.1.1/SnapmakerDevTeam.SnapmakerLuban.installer.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/s/SnapmakerDevTeam/SnapmakerLuban/4.1.1/SnapmakerDevTeam.SnapmakerLuban.installer.yaml:16]: Oops, didn't see this.

ia32 is 32-bit

cc @Trenly

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/s/SnapmakerDevTeam/SnapmakerLuban/4.1.1/SnapmakerDevTeam.SnapmakerLuban.installer.yaml (hunk 2 above)
Comment (Trenly) [reply manifests/s/SnapmakerDevTeam/SnapmakerLuban/4.1.1/SnapmakerDevTeam.SnapmakerLuban.installer.yaml:16]: Thanks, I missed that too 🤣

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**